onnx_caffe2/frontend.py

```diff
@@ -27,7 +24,8 @@
 _special_translator = {}
 
 # caffe2 arguments that needs to be removed
-_blacklist_caffe2_args = {'order', 'global_pooling'}
+_blacklist_caffe2_args = {'order', 'global_pooling',
+                          'cudnn_exhaustive_search', 'use_cudnn'}
```

Comment:
These two attributes are in all caffe2 "brew" generated models, but in onnx we don't have them (and probably will never have them).